images/win/windows2016.json

```diff
@@ -154,11 +154,10 @@
                 3010
             ],
             "scripts": [
-                "{{ template_dir }}/scripts/Installers/Install-KubernetesTools.ps1",
                 "{{ template_dir }}/scripts/Installers/Install-VS.ps1",
+                "{{ template_dir }}/scripts/Installers/Install-KubernetesTools.ps1",
```

Comment:
Can we move docker installation here as well? Or it requires a reboot?

Reply:
This section is run with extra permissions:
```
"elevated_user": "{{user `install_user`}}",
"elevated_password": "{{user `install_password`}}"
```
I don't think that Docker tools require them.

Honestly, I don't think that even KubernetesTools require them but keeping this way to avoid silent breakage

Reply:
ok, let's keep that in mind for the further improvements 😃